content/docs/repository_status/_index.md

```diff
@@ -3,6 +3,20 @@ weight: 7
 title: "Repository Status"
 ---
 
+# CI Dashboard
+
+ 
+
+| Repository | CI Status | Coverage |
+| --- | --- | --- |
+| FAIR-CLI | [![FAIR Data Pipeline CLI](https://github.com/FAIRDataPipeline/FAIR-CLI/actions/workflows/fair-cli.yaml/badge.svg?branch=main)](https://github.com/FAIRDataPipeline/FAIR-CLI/actions/workflows/fair-cli.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/FAIR-CLI/branch/dev/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/FAIR-CLI) |
+| data-registry | [![FAIR Data Registry](https://github.com/FAIRDataPipeline/data-registry/actions/workflows/fair-data-registry.yaml/badge.svg)](https://github.com/FAIRDataPipeline/data-registry/actions/workflows/fair-data-registry.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/data-registry/branch/main/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/data-registry) |
+| cppDataPipeline| [![FDP C++ API](https://github.com/FAIRDataPipeline/cppDataPipeline/actions/workflows/fdp_cpp_api.yaml/badge.svg)](https://github.com/FAIRDataPipeline/cppDataPipeline/actions/workflows/fdp_cpp_api.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/cppDataPipeline/branch/main/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/cppDataPipeline) |
+| DataPipeline.jl | [![Package Testing](https://github.com/FAIRDataPipeline/DataPipeline.jl/actions/workflows/testing.yaml/badge.svg)](https://github.com/FAIRDataPipeline/DataPipeline.jl/actions/workflows/testing.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/DataPipeline.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/DataPipeline.jl) |
+| javaDataPipeline | [![test with reg](https://github.com/FAIRDataPipeline/javaDataPipeline/actions/workflows/build-test-with-registry.yml/badge.svg)](https://github.com/FAIRDataPipeline/javaDataPipeline/actions/workflows/build-test-with-registry.yml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/javaDataPipeline/branch/main/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/javaDataPipeline) |
+| pyDataPipeline | [![pyDataPipeline](https://github.com/FAIRDataPipeline/pyDataPipeline/actions/workflows/pyDataPipeline.yaml/badge.svg?branch=dev)](https://github.com/FAIRDataPipeline/pyDataPipeline/actions/workflows/pyDataPipeline.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/pyDataPipeline/branch/dev/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/pyDataPipeline) |
+| rDataPipeline | [![rDataPipeline](https://github.com/FAIRDataPipeline/rDataPipeline/workflows/build/badge.svg?=1)](https://github.com/FAIRDataPipeline/rDataPipeline/actions) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/rDataPipeline/branch/main/graph/badge.svg?token=xTFk0581AY)](https://codecov.io/gh/FAIRDataPipeline/rDataPipeline) |
```

Comment:
The code coverage badges shouldn't need the token.

Reply:
Thanks!